### PR TITLE
dev, v3.0: add a description for view

### PR DESCRIPTION
**Pull request by XuHuaiyu**
### What is changed, added or deleted? 
Add a description document for VIEW.

### Which version does your change affect? 

dev version
and also need to be copied to v3.0 version.

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by XuHuaiyu** [comment]
@lilin90 PTAL

File: dev/reference/sql/sql-mode.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ TiDB 启动之前采用修改 `--sql-mode="modes"` 配项设置 SQL 模式。
 
 TiDB 启动之后采用 `SET [ SESSION | GLOBAL ] sql_mode='modes'`设置 SQL 模式。设置 GLOBAL 级别的 SQL 模式时用户需要有 SUPER 权限，并且只会影响到从设置 SQL 模式开始后续新建立的连接（注：老连接不受影响)。 SESSION 级别的 SQL 模式的变化只会影响当前的客户端。
 
-Modesis是用逗号 (', ') 间隔开的一系列不同的模式。使用 `SELECT @@sql_mode` 语句查询当前 SQL 模式，SQL 模式默认值：""。
+Modes 是用逗号 (', ') 间隔开的一系列不同的模式。使用 `SELECT @@sql_mode` 语句查询当前 SQL 模式，SQL 模式默认值：""。
```
Comment (lilin90) [inline dev/reference/sql/sql-mode.md]:
```suggestion
Modes 是用逗号 (`,`) 间隔开的一系列不同的模式。使用 `SELECT @@sql_mode` 语句查询当前 SQL 模式，SQL 模式的默认值为 `""`。
```

File: dev/reference/sql/view.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+title: View
```
Comment (lilin90) [inline dev/reference/sql/view.md]: Let's keep this `title` consistent with the 1st-level heading.

```suggestion
title: 视图
```

File: dev/reference/sql/view.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+category: reference
+---
+
+# 视图
+
+TiDB 支持视图，视图是一张虚拟表，该虚拟表的结构由创建视图时的 SELECT 语句定义。使用视图一方面可以对用户只暴露安全的字段及数据，进而保证底层表的敏感字段及数据的安全。另一方面，将频繁出现的复杂查询定义为视图，可以使复杂查询更加简单便捷。
```
Comment (lilin90) [inline dev/reference/sql/view.md]:
```suggestion
TiDB 支持视图，视图是一张虚拟表，该虚拟表的结构由创建视图时的 `SELECT` 语句来定义。使用视图一方面可以对用户只暴露安全的字段及数据，进而保证底层表的敏感字段及数据的安全。另一方面，将频繁出现的复杂查询定义为视图，可以使复杂查询更加简单便捷。
```

File: dev/reference/sql/view.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## 查询视图
+
+查询一个视图和查询一张普通表类似。但是 TiDB 在真正执行查询视图时，会将视图展开成创建视图时定义的 SELECT 语句，进而执行展开后的查询语句。
```
Comment (lilin90) [inline dev/reference/sql/view.md]:
```suggestion
查询一个视图和查询一张普通表类似。但是 TiDB 在真正执行查询视图时，会将视图展开成创建视图时定义的 `SELECT` 语句，进而执行展开后的查询语句。
```

File: dev/reference/sql/view.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+## 样例
+
+下例创建一个视图，并在该视图上进行查询，最后删除该视图。
```
Comment (lilin90) [inline dev/reference/sql/view.md]:
```suggestion
以下例子将创建一个视图，并在该视图上进行查询，最后删除该视图。
```

File: dev/reference/sql/view.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+``` sql
```
Comment (lilin90) [inline dev/reference/sql/view.md]:
```suggestion
```sql
```

File: dev/reference/sql/view.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
++------+
+3 rows in set (0.00 sec)
+
+tidb> drop view v;
+Query OK, 0 rows affected (0.02 sec)
+```
+
+## 局限性
+
+目前 TiDB 中视图有以下局限性：
```
Comment (lilin90) [inline dev/reference/sql/view.md]:
```suggestion
目前 TiDB 中的视图有以下局限性：
```

File: dev/reference/sql/view.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+- 不支持物化视图。
+- TiDB 中视图为只读视图，不支持对视图进行 UPDATE、INSERT、DELETE 等写入操作。
```
Comment (lilin90) [inline dev/reference/sql/view.md]:
```suggestion
- TiDB 中视图为只读视图，不支持对视图进行 `UPDATE`、`INSERT`、`DELETE` 等写入操作。
```

File: dev/reference/sql/view.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+- 对已创建的视图仅支持一种 DDL 操作，即 DROP [VIEW | TABLE]。
```
Comment (lilin90) [inline dev/reference/sql/view.md]:
```suggestion
- 对已创建的视图仅支持一种 DDL 操作，即 `DROP [VIEW | TABLE]`。
```

File: dev/reference/sql/view.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+## 扩展阅读
+
+- [创建试图](/dev/reference/sql/statements/create-view.md)
```
Comment (lilin90) [inline dev/reference/sql/view.md]:
```suggestion
- [创建试图](/reference/sql/statements/create-view.md)
```

File: dev/reference/sql/view.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+- [删除视图](/dev/reference/sql/statements/drop-view.md)
```
Comment (lilin90) [inline dev/reference/sql/view.md]:
```suggestion
- [删除视图](/reference/sql/statements/drop-view.md)
```

File: dev/reference/sql/sql-mode.md [hunk 12]
```diff
@@ -11,7 +11,7 @@ TiDB 启动之前采用修改 `--sql-mode="modes"` 配项设置 SQL 模式。
 
 TiDB 启动之后采用 `SET [ SESSION | GLOBAL ] sql_mode='modes'`设置 SQL 模式。设置 GLOBAL 级别的 SQL 模式时用户需要有 SUPER 权限，并且只会影响到从设置 SQL 模式开始后续新建立的连接（注：老连接不受影响)。 SESSION 级别的 SQL 模式的变化只会影响当前的客户端。
 
-Modesis是用逗号 (', ') 间隔开的一系列不同的模式。使用 `SELECT @@sql_mode` 语句查询当前 SQL 模式，SQL 模式默认值：""。
+Modes 是用逗号 (',') 间隔开的一系列不同的模式。使用 `SELECT @@sql_mode` 语句查询当前 SQL 模式，SQL 模式默认值：`""`。
```
Comment (zz-jason) [inline dev/reference/sql/sql-mode.md]: 这个默认值不对吧

File: dev/reference/sql/view.md [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+Query OK, 0 rows affected (0.02 sec)
+```
+
+## 局限性
+
+目前 TiDB 中的视图有以下局限性：
+
+- 不支持物化视图。
+- TiDB 中视图为只读视图，不支持对视图进行 `UPDATE`、`INSERT`、`DELETE` 等写入操作。
+- 对已创建的视图仅支持一种 DDL 操作，即 `DROP [VIEW | TABLE]`。
```
Comment (zz-jason) [inline dev/reference/sql/view.md]:
```suggestion
- 对已创建的视图仅支持 `DROP` 的 DDL 操作，即 `DROP [VIEW | TABLE]`。
```

**Pull request merged**